apps/web/src/scenes/CommunityPage/CommunityPageViewHeader.tsx

```diff
@@ -198,18 +230,50 @@ export default function CommunityPageViewHeader({ communityRef, queryRef }: Prop
             onClick={handleShowMoreClick}
           />
         )}
+
+        <StyledEditIconWrapper>
+          <EditPencilIcon />
+        </StyledEditIconWrapper>
+
+        <StyledTooltip
```

Comment:
I think we only want to show the tooltip if you hover on the pencil icon

![CleanShot 2024-02-02 at 21 13 18@2x](https://github.com/gallery-so/gallery/assets/80802871/a5f35160-2623-4e81-a8a9-c6281ee7e88c)
![CleanShot 2024-02-02 at 21 13 14@2x](https://github.com/gallery-so/gallery/assets/80802871/a8cfc313-3dd9-48b3-82d4-da291f9b8bb1)

Reply:
so it looks like it should be:

hover on editable area: show pencil icon
hover on pencil icon: show tooltip

Reply:
lets make the tooltip appear on the right of the icon too, right now it's on top

oh also for the icon, we have a `IconContainer` component that maybe you can try - it has the tooltip built in, and also has a `tooltipPlacement` prop that lets you specify which direction it appears in!

Reply:
Thanks for tips! Refactor it to use the `IconContainer` and its working great. The code much cleaner too now 🫠


https://github.com/gallery-so/gallery/assets/4480258/0b283dfa-c1e3-4778-ba40-4b6070311b3e